reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md

```diff
@@ -223,20 +229,27 @@ Accept wildcard characters: False
 
 ### CommonParameters
 
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
 
 ## INPUTS
 
 ### System.Management.Automation.PSObject
 
-You can pipe a DifferenceObject object to Compare-Object.
+You can pipe a **DifferenceObject** object to this cmdlet.
 
 ## OUTPUTS
 
-### None, or the objects that are different
+### None
+
+If the objects are the same, nothing is returned.
+
+### System.Management.Automation.PSCustomObject
 
-When you use the PassThru parameter, Compare-Object returns the objects that differed.
-Otherwise, this cmdlet does not generate any output.
+If the objects are different, `Compare-Object` wraps the differing objects in a `PSCustomObject` wrapper with a `SideIndicator` property to reference the differences.
+
+### Other
+
+When you use the *PassThru* parameter, **Compare-Object** omits the `PSCustomObject` wrapper around the compared objects and returns the differing objects, unchanged.
```

Comment:
This should text should also appear as the description of the -PassThru parameter.

Also, the formatting of this is all wrong. Per the style guide:

- When talking about a cmdlet in paragraph, use (`) to highlight cmdlet names
- Property names and parameter names should be in bold